RULES_DESCRIPTIONS.md

```diff
@@ -837,13 +865,13 @@ _Configuration_: N/A
 
 Example:
 
-```go
+```golang
```

Comment:
I don't understand why you used golang, it's not the canonical name, and this one is not in the mdsf config

Reply:
I explained in the PR's description.

Reply:
Indeed, I didn't see it